src/test/java/io/github/perrymant/moneymaker/file_with_data

```diff
@@ -0,0 +1,2 @@
+2018/01/01, CREDIT, 200, 200, Got paid
```

Comment:
Should these files even be committed to Git if we only want to make sure we can read and write files? I suggest putting them in a `src/test/resources/dump/` folder, and then adding this folder to your `.gitignore`